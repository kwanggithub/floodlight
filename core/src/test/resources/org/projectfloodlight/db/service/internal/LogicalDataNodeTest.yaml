treespaces:
  - name: LogicalDataNodeTest
    data_sources:
        - name: ds1
          implementation_class: org.projectfloodlight.db.data.memory.MemoryDataSource
          config: true
        - name: ds2
          implementation_class: org.projectfloodlight.db.data.memory.MemoryDataSource
          config: true
        - name: ds3
          implementation_class: org.projectfloodlight.db.data.memory.MemoryDataSource
          config: false
    data_source_mappings:
      - data_source: "$data-source"
    module_search_paths:
      - path: src/test/resources/org/projectfloodlight/db/service/internal
        recursive: true
    modules:
      - name: LogicalDataNodeTest
