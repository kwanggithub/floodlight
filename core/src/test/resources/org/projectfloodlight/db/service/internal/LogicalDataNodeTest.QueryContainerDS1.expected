{
  "child1" : 5,
  "child4" : {
    "child4.2" : 100
  }
}